pylti/tests/test_common.py

```diff
@@ -76,7 +78,8 @@ def test_ltioauthdatastore(self):
         self.assertIsNone(store.lookup_consumer("keyNS"))
         self.assertIsNone(store.lookup_cert("keyNS"))
 
-    def test_ltioauthdatastore_no_consumers(self):
+    def test_lti_oauth_data_store_no_consumers(self):
+        # pylint: disable=invalid-name
```

Comment:
So I think for these long names we can just create a pylintrc file that allows for longer names.  Thus far the entire contents of the rc file would be:
```ini
[BASIC]

# Regular expression which should only match correct method names
method-rgx=[a-z_][a-z0-9_]{2,40}$
```
Where you will need to adjust the `40` in `{2,40}` for the right length of method names as longs ones are fine